contributing-guides/maintainers-guide.md

```diff
@@ -58,20 +58,18 @@ as a guideline for current and future maintainers.
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
   (2) have the **review comments addressed**, and
   (3) get **approved reviews by two maintainers**
-  (the second maintainer can perform the merge immediately after accepting.)
+  (4) are at least **24 hours** old unless changes are trivial
```

Comment:
I think 24 hours is a good rule of thumb. I'd rephrase this a little though.
```suggestion
  (2) have the **review comments addressed**,
  (3) get **approved reviews by two maintainers**, and
  (4) have been open for at least **24 hours** unless the changes are trivial
```

Reply:
Looks like Sbrl keep the explanation that
> the second maintainer can perform the merge immediately after accepting.
Leaving explicit that after the 2nd approve the maintainer shouldn't wait another stuff.

Reply:
@sbrl reffering to your comment [here](https://github.com/tldr-pages/tldr/pull/6397#discussion_r693549872), what do you think about this:
```suggestion
  (2) have the **review comments addressed**,
  (3) get **approved reviews by two maintainers**, (the second maintainer can merge immediately after approving) and
  (4) have been open for at least **24 hours** unless the changes are trivial
```

Reply:
@CleanMachine1 could you please apply this? ⬆️

Reply:
@marchersimon Could you please resend the suggestion or feel free to edit the page, I can't see whats being changed.

Reply:
Done in ba415df